src/xunit.performance.api/XunitPerformanceHarness.cs

```diff
@@ -160,5 +214,10 @@ private void FreeUnManagedResources()
         private readonly List<string> _outputFiles;
         private readonly List<string> _typeNames;
         private bool _disposed;
+        public delegate void SetupDelegate(PerformanceTestConfig config, ProcessStartInfo processStartInfo);
+        public delegate void PostIterationDelegate(PerformanceTestConfig config);
+        public delegate ScenarioBenchmark TeardownDelegate(PerformanceTestConfig config);
```

Comment:
Ah... I just commented above that we should use these instead of Action/Func.  :)